ardupilot/equipment/power/20010.BatteryContinuous.uavcan

```diff
@@ -0,0 +1,33 @@
+#
+# Battery data to be sent continuously
+#
+
+float16 temperature_cells       # [C]  : Pack mounted thermistor (preferably installed between cells), NAN: field not provided
+float16 temperature_pcb         # [C]  : Battery PCB temperature (likely output FET(s) or current sense resistor), NAN: field not provided
+float16 temperature_other       # [C]  : Application dependent, NAN: field not provided
+float32 current                 # [A]  : Positive: defined as a discharge current. Negative: defined as a charging current, NAN: field not provided
+float16 voltage                 # [V]  : Battery voltage
+uint8 state_of_charge           # [%]  : The estimated state of charge, in percent remaining (0 - 100).
```

Comment:
We are using 22B of payload bytes currently. This requires 4 CAN frames (5 + 7 + 7 +7) meaning we have 4B free that are wasted. 

The change below uses 3 of those bytes somewhat usefully? I'm open to any other suggestions?  

```suggestion
float32 voltage                 # [V]  : Battery voltage
float16 state_of_charge           # [%]  : The estimated state of charge, in percent remaining (0 - 100).
```